resources/ctia-default.properties

```diff
@@ -146,3 +146,4 @@ ctia.hook.kafka.topic.name=ctia-events
 ctia.hook.kafka.topic.num-partitions=1
 ctia.hook.kafka.topic.replication-factor=1
 
+ctia.versions.config=local
```

Comment:
I used dev in IROH

```suggestion
ctia.versions.config=dev
```

Reply:
I prefer to keep `local` if the configuration is not set in production